---a/net/minecraft/enchantment/EnchantmentOxygen.java
+++b/net/minecraft/enchantment/EnchantmentOxygen.java
@@ -2,29 +2,25 @@
 
 import net.minecraft.util.ResourceLocation;
 
-public class EnchantmentOxygen extends Enchantment
-{
+public class EnchantmentOxygen extends Enchantment {
+
     private static final String __OBFID = "CL_00000120";
 
-    public EnchantmentOxygen(int enchID, ResourceLocation p_i45766_2_, int p_i45766_3_)
-    {
+    public EnchantmentOxygen(int enchID, ResourceLocation p_i45766_2_, int p_i45766_3_) {
         super(enchID, p_i45766_2_, p_i45766_3_, EnumEnchantmentType.ARMOR_HEAD);
         this.setName("oxygen");
     }
 
-    public int getMinEnchantability(int enchantmentLevel)
-    {
+    public int getMinEnchantability(int enchantmentLevel) {
         return 10 * enchantmentLevel;
     }
 
-    public int getMaxEnchantability(int enchantmentLevel)
-    {
+    public int getMaxEnchantability(int enchantmentLevel) {
         return this.getMinEnchantability(enchantmentLevel) + 30;
     }
 
-    public int getMaxLevel()
-    {
+    public int getMaxLevel() {
         return 3;
     }
-}
+}
